### DISCUSSION TITLE
Can I make the changelog videos?

### DISCUSSION THREAD

**Discussion by Secret-chest** [ScratchAddons/ScratchAddons#6038] [Discussion]
CS is no longer making any videos, and I'll be happy to make them myself.

@WorldLanguages @griffpatch

**Comment by WorldLanguages** [comment]
https://github.com/ScratchAddons/ScratchAddons/pull/5999#issuecomment-1535008892

**Reply by Secret-chest** [reply]
I'd like to first make a fake video and post it here, and then we'll talk.

**Reply by WorldLanguages** [reply]
Everyone is welcome to post "fake videos". I was trying to say that it would be inconvenient not to use Discord for real videos.
